assets/source/setup-guide/app/steps/SetupTracking.js

```diff
@@ -329,10 +373,19 @@ const SetupTracking = ( { view } ) => {
 								</Button>
 							</>
 						}
-						link={
-							wcSettings.pinterest_for_woocommerce.pinterestLinks
-								.SetupTracking
-						}
+						readMore={ {
+							href:
+								wcSettings.pinterest_for_woocommerce
+									.pinterestLinks.SetupTracking,
+							onClick: () =>
+								recordEvent( 'pfw_documentation_link_click', {
+									link_id: 'setup-tracking',
+									context: view,
+									href:
+										wcSettings.pinterest_for_woocommerce
+											.pinterestLinks.SetupTracking,
```

Comment:
📝
The property `SetupTracking` doesn't exist, and I think it may have been a problem from [the beginning](https://github.com/woocommerce/pinterest-for-woocommerce/commit/b19c441e4591cc623b689bb55ff0c329184b9621#diff-39ee87358882006b4b222b52f6a4cbf74df2e0267b947b808ca7f4c16dc3f08cR73). I will create an issue for it later.

Reply:
Opened #262.